packages/dds/sequence/src/test/sharedIntervalCollection.spec.ts

```diff
@@ -101,8 +101,8 @@ describe("SharedIntervalCollection", () => {
 		});
 
 		it("Can add intervals from multiple clients", () => {
-			collection1.add(0, 20, IntervalType.Simple);
```

Comment:
Do we want to keep coverage for these interval types until they are actually removed since we still support them until then?